packages/db/src/core/cli/commands/execute/index.ts

```diff
@@ -16,6 +15,7 @@ import {
 import { bundleFile, importBundledFile } from '../../../load-file.js';
 import type { DBConfig } from '../../../types.js';
 import { getManagedRemoteToken } from '../../../utils.js';
+import { isDbError } from '../../../../runtime/virtual.js';
```

Comment:
I don't know if this causes the CI fail, but I _think_ we may need to copy `isDbError` into `../../../utils.js` so there's two implementations of it for now. The `virtual.js` is only for the `astro:db` virtual module and we don't want to mix it with the non-virtual code.